server/plugin.go

```diff
@@ -526,6 +530,8 @@ func (p *Plugin) OnSendDailyTelemetry() {
 			server++
 		case CloudInstanceType:
 			cloud++
+		case CloudOAuthInstanceType:
+			cloud++
```

Comment:
In order to track MM servers that are using this new instance type, I think we should a separate category here `cloud_oauth_instance_count`